recipes/fmt/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,18 @@
+from conans import ConanFile, CMake, tools
```

Comment:
```suggestion
# pylint: skip-file
from conans import ConanFile, CMake, tools
```

For simplicity's sake, I feel it is easier to write this line here to skip the linter than to modify the GH action or the logic. Eventually, this file will be removed, and this comment will be gone together with the file.